utt/privacy-wallet-service/proto/api/v1/wallet-api.proto

```diff
@@ -25,20 +25,54 @@ message PrivacyAppConfigResponse {
     bytes configuration = 1; // Blob of privacy configuration data
 }
 
+message PrivacyWalletConfigRequest {
+    string private_key = 1;
+    string public_key = 2;
+    bytes public_application_config = 3;
+    string storage_path = 4;
+}
+
+message PrivacyWalletConfigResponse {
+    bool succ = 1;
+}
+
+message UserRegistrationRequest {
+}
+
+message UserRegistrationResponse {
+    bytes rcm1 = 1;
+    bytes rcm1_sig = 2;
+}
+
+message UserRegistrationUpdateRequest {
+    bytes rcm_sig = 1;
+    repeated uint64 s2 = 2;
+}
+
+message UserRegistrationUpdateResponse {
+    bool succ = 1;
```

Comment:
At the moment I prefer to have a response per request, it will make it easier to debug and make sure the correct request was passed to the wallet service